README.md

```diff
@@ -31,6 +31,19 @@ The elements shown in the architecture diagram above are deployed by the [Cyral
 
 * Make sure you have access to your AWS environment with an account that has sufficient permissions to deploy the sidecar. The minimum permissions must allow for the creation of the elements listed previously. We recommend Administrator permissions (`AdministratorAccess` policy) as the module creates an IAM role.
 
+#### Certificates Needed for the Sidecar
+
+This deployment looks for two secrets to be created prior to deployment:
+
+* Sidecar CA certificate
+* Sidecar self-signed certificate
+
+These certificates are used to terminate TLS connections between the client and sidecar as well as to sign requests that are proxied through the sidecar. These certificates should be from a trusted certificate authority. 
+
+Please refer to the [Advanced](#advanced) section of this guide on deploying the certificates.
+
+> **NOTE ::** These certificates must be created before deployment or your deploy will fail.
+
```

Comment:
Make this into a simplified bullet in the `Requirements` section and point to the actual page in the `Advanced` section.